web-server/src/components/Service/SystemStatus.tsx

```diff
@@ -0,0 +1,167 @@
+    };
+  }, [dispatch]);
+
+  const { addPage } = useOverlayPage();
+
+  const ServiceTitle: { [key: string]: string } = {
+    [ServiceNames.API_SERVER]: 'Backend Server',
+    [ServiceNames.REDIS]: 'Redis Database',
+    [ServiceNames.POSTGRES]: 'Postgres Database',
+    [ServiceNames.SYNC_SERVER]: 'Sync Server'
+  };
+  return (
+    <FlexBox col gap={2} sx={{ padding: '16px' }}>
+      <Line bold white fontSize="24px" sx={{ mb: 2 }}>
+        System Status
+      </Line>
+      <Divider sx={{ mb: 2, backgroundColor: 'rgba(255, 255, 255, 0.2)' }} />
+
+      {loading ? (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center'
+          }}
+        >
+          <CircularProgress color="primary" size={50} />
+        </Box>
+      ) : (
+        <FlexBox col gap={2}>
+          {Object.keys(services).map((serviceName) => {
+            const serviceKey = serviceName as ServiceNames;
+            const { isUp } = services[serviceKey];
+            return (
+              <CardRoot
+                key={serviceName}
+                onClick={() => {
+                  addPage({
+                    page: {
+                      ui: 'system_logs',
+                      title: `${ServiceTitle[serviceName]} Logs`,
+                      props:{
+                          serviceName:serviceName
+                      }
+                    }
+                  });
+                }}
+                sx={{
+                  transition: 'box-shadow 0.2s ease',
+                  '&:hover': {
+                    boxShadow: '0 4px 15px rgba(0, 0, 0, 0.1)'
+                  },
+                  backgroundColor: 'rgba(255, 255, 255, 0.05)',
+                  borderRadius: '12px',
+                  border: `1px solid ${
+                    isUp ? 'rgba(0, 255, 0, 0.3)' : 'rgba(255, 0, 0, 0.3)'
+                  }`,
+                  padding: '16px',
+                  cursor: 'pointer',
+                  position: 'relative',
+                  overflow: 'hidden'
+                }}
+              >
+                <FlexBox col flexGrow={1} minHeight="5em">
+                  <FlexBox alignCenter justifyContent="space-between">
+                    <Line
+                      white
+                      bold
+                      sx={{
+                        fontSize: '1.2em',
+                        display: 'flex',
+                        alignItems: 'center'
+                      }}
+                    >
+                      {ServiceTitle[serviceName]}
+                      <Box
+                        component="span"
+                        sx={{
+                          display: 'inline-block',
+                          marginLeft: '6px',
+                          width: '10px',
+                          height: '10px',
+                          borderRadius: '50%',
+                          backgroundColor: isUp ? '#28a745' : '#dc3545'
```

Comment:
i am assuming this is the online-offline dot on the right of log-title

we have some theme colors inbuilt. 

search for how we use `theme.colors.success.main` and `theme.colors.error.main` in our code-base.

you won't have to create your own colors

Reply:
Great thanks for letting me know , we can use similar colors for log as well but I was thinking the colors are useful because in CLI we have a lot of logs at once but in this case we only show log for one service at a time does these make sense in here ?

Reply:
sorry, i don't understand your comment, re-explain?

the colors that you hardcoded (green and red) can be replaced with our own thematic colors of `success` and `error`. that's all i wanted you to change

Reply:
Got you eshan sorry I miss understood